libs/execution_context/ebpf_program.h

```diff
@@ -146,17 +135,38 @@ extern "C"
     ebpf_program_invoke(_In_ const ebpf_program_t* program, _In_ void* context, _Out_ uint32_t* result);
 
     /**
-     * @brief Get the address of a helper function.
+     * @brief Store the helper function IDs that are used by the eBPF program in an array inside the program object. The
+     array index is the helper function ID to be used by uBPF whereas the array value is the actual helper ID.
```

Comment:
```suggestion
     * @brief Store the helper function IDs that are used by the eBPF program in an array
     *  inside the program object. The array index is the helper function ID to be used by
     *  uBPF whereas the array value is the actual helper ID.
```
(start each comment line with `*`)